mvpa2/cmdline/cmd_ttest.py

```diff
@@ -58,14 +61,35 @@ def setup_parser(parser):
                         belong to.  E.g. if values are accuracies, it would be the
                         'greater', if errors -- the 'less'""")
 
+def check_filetype(fn):
+    if '.nii' in fn:
```

Comment:
Ah, II see the point for having a function now... So make it a guess_backend which would return hdf or nibabel ;-)